tests/network/vmi_networking.go

```diff
@@ -497,7 +497,7 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 			By("checking eth0 Pci address")
 			testVMI := libvmi.NewAlpine()
 			tests.AddExplicitPodNetworkInterface(testVMI)
-			testVMI.Spec.Domain.Devices.Interfaces[0].PciAddress = "0000:81:00.1"
+			testVMI.Spec.Domain.Devices.Interfaces[0].PciAddress = "0000:01:00.0"
```

Comment:
Why is it required to change the pci address? What was the problem with the old one?

Reply:
It does not appear in the guest. I guess the bus is our out of range.
Libvirt is silently ignoring it.